srcpkgs/diffuse/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'diffuse'
+pkgname=diffuse
+_dist_ver=0.5.0
+_dist_rel=beta.1
+version="${_dist_ver}${_dist_rel}"
+revision=1
+archs=noarch
+wrksrc="${pkgname}-${_dist_ver}-${_dist_rel}"
+depends="python3-gobject"
+short_desc="Graphical tool for merging and comparing text files (Python3-fork)"
+maintainer="Dima Burmistrov <dmabrm@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/MightyCreak/diffuse"
+distfiles="https://github.com/MightyCreak/diffuse/archive/v${_dist_ver}-${_dist_rel}.tar.gz"
+checksum=00f62302e0bad049b99dbc8f9c12b2c6eb10046489f52a11069dcb326d3f41fb
+python_version=3
+
+do_install() {
+	cp -av src/* "${DESTDIR}/"
```

Comment:
There is a 0.6.0 release, and it requires a bit more complicated setup than this. I believe `build_style=meson` will be necessary.